packages/angular/package.json

```diff
@@ -10,17 +10,17 @@
 	},
 	"private": true,
 	"dependencies": {
-		"@angular/animations": "~12.1.0",
+		"@angular/animations": "^12.2.16",
 		"@angular/cdk": "^11.2.9",
-		"@angular/common": "~12.1.0",
-		"@angular/compiler": "~12.1.0",
-		"@angular/core": "~12.1.0",
-		"@angular/forms": "~12.1.0",
-		"@angular/platform-browser": "~12.1.0",
-		"@angular/platform-browser-dynamic": "~12.1.0",
-		"@angular/router": "~12.1.0",
-		"devextreme": "21.2.7",
-		"devextreme-angular": "21.2.7",
+		"@angular/common": "^12.2.16",
+		"@angular/compiler": "^12.2.16",
+		"@angular/core": "^12.2.16",
+		"@angular/forms": "^12.2.16",
+		"@angular/platform-browser": "^12.2.16",
+		"@angular/platform-browser-dynamic": "^12.2.16",
+		"@angular/router": "^12.2.16",
+		"devextreme": "22.1.5",
+		"devextreme-angular": "22.1.5",
```

Comment:
Why do we use the frozen version? Could we you use the version with ^

Reply:
fixed